markdown/3.x/en/ff-sortbox.md

```diff
@@ -3,9 +3,22 @@ You can choose from two different kinds of sortboxes. The `ff-sortbox-select` is
 The `ff-sortbox` is a custom-built dropdown element. Both can be used with or without customization.
 
 ## Native HTML select
-The code below will display a native HTML `select`. The `select`-element as well as the `option`-element are optional. It will default to `<select><option>{{description}}</option></select>` if not present.
+The `ff-sortbox-select` displays a native HTML `select` element.
 
-```html
+Minimal setup:
+```
+<ff-sortbox-select></ff-sortbox-select>
+```
+
+Specifying only the `option` element:
+```
+<ff-sortbox-select>
+  <option>{{description}}</option>
```

Comment:
While at it, let's use 4-space indentation to be consistent with the other examples.